doc/source/_toc.yml

```diff
@@ -63,25 +63,29 @@ parts:
             title: "Getting Started"
           - file: train/key-concepts
             title: "Key Concepts"
-          - file: train/user-guides
-            title: "User Guides"
+          - file: train/distributed-pytorch
```

Comment:
nit: I'm not sure where this is configured, but would be good to have the drop-down caret show in the ToC.

Reply:
I'd like that, too, and have no idea why it doesn't show up. I'll try a few more things...

Reply:
I took a look and think it's defined here: https://github.com/ray-project/ray/blob/672ce553ca1cd07896438eb0d0abe914b349023d/doc/source/_static/js/custom.js#L32-L55

Reply:
Thank you! I'll update the list